api/controllers/models/agency.js

```diff
@@ -1,9 +1,9 @@
 class Agency {
     constructor(res) {
```

Comment:
same here. inconsistent use of semicolons.

Reply:
removed, should run eslint on all files to remove may be later.